app/src/links.js

```diff
@@ -14,11 +14,6 @@ module.exports = {
     site: {
         network: formatExternalUrl(streamrRoot, 'learn/network'),
     },
-    faq: 'https://www.streamr.com/faq',
-    aboutUs: 'https://www.streamr.com/about',
-    howItWorks: 'https://www.streamr.com#howItWorks',
-    streamrSystem: 'https://www.streamr.com#streamrSystem',
-    tryTheEditor: 'https://www.streamr.com#tryTheEditor',
     contributionSummary: 'https://s3.amazonaws.com/streamr-public/Crowdcontribution+Information+Summary+(20170912).pdf',
```

Comment:
Email addresses in this file still with `streamr.com` domain:
```
     contact: {
        general: 'mailto:contact@streamr.com',
        media: 'mailto:media@streamr.com',
        jobs: 'mailto:jobs@streamr.com',
        labs: 'mailto:labs@streamr.com',
    },
```
Also in `definitions.json`:
```
    "contactGeneral": "mailto:contact@streamr.com",
    "contactJobs": "mailto:jobs@streamr.com",
    "contactLabs": "mailto:labs@streamr.com",
    "contactMedia": "mailto:media@streamr.com",
```

Reply:
As stated in [the ticket](https://streamr.atlassian.net/browse/PLATFORM-1076):
> _**Do not** update email addresses, as streamr.network emails don't work yet._